services/openshift-service-impl/src/test/java/org/kontinuity/catapult/service/openshift/impl/OpenShiftServiceTestBase.java

```diff
@@ -27,8 +33,25 @@
 
     private final Collection<OpenShiftProject> createdProjects = new ArrayList<>();
 
+    private final URI pipelineTemplate;
+    //private final InputStream pipelineTemplate = Thread.currentThread().getContextClassLoader().getResourceAsStream("test-template.yml");
+    
     protected abstract OpenShiftService getOpenShiftService();
     
+	public OpenShiftServiceTestBase() {
```

Comment:
Indeed, there was actually no justification for doing this initialization in the constructor. I moved it into the `setup` method.